### PR TITLE
Standardize OCR outputs and plot them

### PR DESCRIPTION
**Pull request by shrav-k**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
+# Linter
+black
+isort
+
+# Testing
+pytest
+
+# Visualization
+matplotlib
```
Comment (CarterMcClellan) [inline requirements.txt]: do we wanna make this a part of the core package? normally idgaf but matplotlib is BIG

File: src/supercontrast/utils/image.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import requests
+
+from io import BytesIO
+from PIL import Image
+from typing import Tuple, Union
+
+
+def get_image_size(image: Union[str, bytes]) -> Tuple[int, int]:
```
Comment (CarterMcClellan) [inline src/supercontrast/utils/image.py:8]: nice change

File: examples/run_examples.sh [hunk 3]
```diff
@@ -1,17 +0,0 @@
-#!/bin/bash
-
-# Azure
```
Comment (CarterMcClellan) [inline examples/run_examples.sh:3]: ah fuck me we are gonna need to squash this can't be in the commit history

Comment (shrav-k) [reply examples/run_examples.sh:3]: yeah I don't know where this got added

File: src/supercontrast/utils/image.py (hunk 2 above)
Comment (shrav-k) [reply src/supercontrast/utils/image.py:8]: :)

File: requirements.txt (hunk 1 above)
Comment (shrav-k) [reply requirements.txt]: removed

File: examples/run_examples.sh (hunk 3 above)
Comment (CarterMcClellan) [reply examples/run_examples.sh:3]: definetly my fault

**Review by CarterMcClellan — APPROVED** [review]

**Pull request merged**